src/dom/dom.js

```diff
@@ -596,7 +596,7 @@ function dom(p5, fn){
    *     'https://p5js.org/assets/img/asterisk-01.png',
```

Comment:
Broken URL; maybe grab an image from another part of the reference that works? eg from https://beta.p5js.org/reference/p5/image/?